audb/core/load.py

```diff
@@ -603,15 +614,28 @@ def _tables(
         deps: Dependencies,
         tables: typing.Optional[typing.Union[str, typing.Sequence[str]]],
 ) -> typing.Sequence[str]:
+
     if tables is None:
-        tables = deps.table_ids
-    elif isinstance(tables, str):
+        return deps.table_ids
+    elif len(tables) == 0:
+        return []
+
+    if isinstance(tables, str):
         pattern = re.compile(tables)
-        tables = []
+        requested_tables = []
         for table in deps.table_ids:
             if pattern.search(table):
-                tables.append(table)
-    return tables
+                requested_tables.append(table)
+    else:
+        requested_tables = tables
+
+    if (
+            len(requested_tables) == 0
+            or any([table not in deps.table_ids for table in requested_tables])
```

Comment:
Same here, maybe we should handle this case separately and list only the tables that are not found?